LoRaEngine/LoraKeysManagerFacade/IDeviceTwin.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
+{
+    using System;
+
+    public interface IDeviceTwin
```

Comment:
I'd like to see `DevEui`, `StationEui` and `DevAddr` being used here rather than degenerating to a `string`.

Also curious to know why `DeviceId` is a property and the others modeled as methods?

Reply:
Hi, 

Since original values are strings. How should I proceed to convert it to DevEui,... ?

For the second point I only kept the code as existing (since it was extensions methods). I'll change that to properties and accessors.